app/templates/teachers/hovers/progress-dot-single-student-level.jade

```diff
@@ -19,9 +19,10 @@ if completed
     if dateCompleted
       span= new Date(dateCompleted).toLocaleString()
     +timePlayed
-  //- .small-details
-  //-   i(data-i18n='teacher.click_to_view_solution')
-  //-     | click to view solution
+  if !isProject && !isLadder
+    .small-details.nowrap
+      //- TODO: data-i18n doesn't work here
```

Comment:
It doesn't? Need me to take a look at why?

Reply:
Not a ship blocker, but you can if you want.  I've spent 20 minutes on this so far, and can't really justify any more at this point.